frontend/nuxt.config.js

```diff
@@ -34,13 +34,12 @@ export default {
             { charset: 'utf-8' },
             { name: 'viewport', content: 'width=device-width, initial-scale=1' },
             { name: 'description', content: SITE_DESCRIPTION },
-            { property: 'og:site_name', content: SITE_TITLE },
+            { property: 'og:title', content: SITE_TITLE },
             { property: 'og:type', content: 'website' },
             { property: 'og:description', content: SITE_DESCRIPTION },
             { property: 'og:image', content: '/preview.jpeg' },
             { name: 'twitter:card', content: 'summary_large_image' },
-            { property: 'og:image:width', content: '1200' },
-            { property: 'og:image:height', content: '630' },
```

Comment:
Why would you delete with and heigh? For some social networks it's essential for showing the image for the first time (at least this was a problem few years ago with facebook)

Reply:
From what I see they are optional now (https://ogp.me/#structured).

The main reason, why I removed them was that Discord thought we were only presenting an image on our website and hence displayed our link, as only the image, without any site description:

![Screenshot 2022-07-21 at 17 46 47](https://user-images.githubusercontent.com/30908158/180257300-e5594545-008a-4a01-b9e1-3a63a898cee3.png)

Which is not what we want. I also followed a tutorial on [reddit](https://www.reddit.com/r/discordapp/comments/82p8i6/a_basic_tutorial_on_how_to_get_the_most_out_of/) where a user outlined which OG properties are used by Discord and how you can use them to customize the link previews.